### ISSUE TITLE
Router: Lack of hooks to perform transformations on route titles

### ISSUE THREAD

**Issue by Vheissu** [aurelia/aurelia#1344]
Quite a common use case in Aurelia 1 routing for me was the use of the Aurelia i18n package to perform text translations on everything in my app, including router titles. The router lacks any kind of hook or ability to run a global intercept to modify and translate the title. We have this behaviour documented and working in v1 routing which effectively works like this:

```
const router = aurelia.container.get(AppRouter);
router.transformTitle = title => i18n.tr(title);
```

This current lack of functionality means it is not on par with v1 routing.

**Comment by Sayan751** [comment]
Closing this as it seems to be a duplicate of #1339 and should be fixed once the PR #1357 is merged.
